sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/autorest.md

```diff
@@ -0,0 +1,108 @@
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: SecurityInsights
+namespace: Azure.ResourceManager.SecurityInsights
+require: https://github.com/Azure/azure-rest-api-specs/blob/53b1affe357b3bfbb53721d0a2002382a046d3b0/specification/securityinsights/resource-manager/readme.md
+tag: package-preview-2022-08
+output-folder: $(this-folder)/Generated
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
+  flatten-payloads: false
+
+request-path-to-parent:
+  /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence/main/indicators: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence/main/indicators/{name}
+
+request-path-to-resource-data:
+  /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence/main/indicators/{name}: ThreatIntelligenceIndicatorModel
+
+operation-positions:
+  ThreatIntelligenceIndicators_List: collection
+
+rename-mapping:
+  ThreatIntelligenceIndicatorModel: ThreatIntelligenceIndicator
+
+override-operation-name:
+  DomainWhois_Get: GetDomainWhoisInformation
+
+format-by-name-rules:
+  'tenantId': 'uuid'
+  'ETag': 'etag'
+  'location': 'azure-location'
+  '*Uri': 'Uri'
+  '*Uris': 'Uri'
+
+rename-rules:
+  CPU: Cpu
+  CPUs: Cpus
+  Os: OS
+  Ip: IP
+  Ips: IPs|ips
+  ID: Id
+  IDs: Ids
+  VM: Vm
+  VMs: Vms
+  Vmos: VmOS
+  VMScaleSet: VmScaleSet
+  DNS: Dns
+  VPN: Vpn
+  NAT: Nat
+  WAN: Wan
+  Ipv4: IPv4|ipv4
+  Ipv6: IPv6|ipv6
+  Ipsec: IPsec|ipsec
+  SSO: Sso
+  URI: Uri
+  Etag: ETag|etag
+
+directive:
+  - rename-operation:
+      from: ThreatIntelligenceIndicator_Get
+      to: ThreatIntelligenceIndicators_Get
+  - rename-operation:
+      from: ThreatIntelligenceIndicator_Create
+      to: ThreatIntelligenceIndicators_Update
+  - rename-operation:
+      from: ThreatIntelligenceIndicator_Delete
+      to: ThreatIntelligenceIndicators_Delete
+  - rename-operation:
+      from: ThreatIntelligenceIndicator_AppendTags
+      to: ThreatIntelligenceIndicators_AppendTags
+  - rename-operation:
+      from: ThreatIntelligenceIndicator_ReplaceTags
+      to: ThreatIntelligenceIndicators_ReplaceTags
+  - rename-operation:
+      from: ThreatIntelligenceIndicator_CreateIndicator
+      to: ThreatIntelligenceIndicators_Create
+  - rename-operation:
+      from: ThreatIntelligenceIndicatorMetrics_List
+      to: ThreatIntelligenceIndicators_ListMetrics
+  - rename-operation:
+      from: ThreatIntelligenceIndicator_QueryIndicators
+      to: ThreatIntelligenceIndicators_Query
+  - rename-operation:
+      from: Bookmark_Expand
+      to: Bookmarks_Expand
```

Comment:
Are these expected? Changing this will make the extra ifnoramtion in our operation's summary inaccurate